libraries/codegen-tooling/inc/can_unpack.h

```diff
@@ -57,7 +57,7 @@
   can_unpack_impl_u16((msg_ptr), 8, (throttle_u16_ptr), (direction_u16_ptr),              \
                       (cruise_control_u16_ptr), (mechanical_brake_state_u16_ptr))
 
-#define CAN_UNPACK_LIGHTS_STATES(msg_ptr, light_id_u8_ptr, light_state_u8_ptr)                     \
+#define CAN_UNPACK_LIGHT_STATE(msg_ptr, light_id_u8_ptr, light_state_u8_ptr)                       \
```

Comment:
@ArshanKhanifar did you manually change these? If so I need to redo this via a re-roll of codegen tooling. You can't just change it here.

Reply:
I just changed it to show you what'd be like, better naming, for this. I know this is generated.